lib/membrane_camera_capture_plugin/camera.ex

```diff
@@ -82,4 +84,14 @@ defmodule Membrane.CameraCapture do
   def handle_info({:frame_provider, _buffer}, _ctx, state) do
     {[], state}
   end
+
+  defp pixel_format_to_atom("yuv420"), do: :I420
```

Comment:
Wasn't it supposed to be "yuv420p"? I420 is a planar format so it would make sense and I didn't even find "yuv420" here: http://ffmpeg.org/doxygen/trunk/pixdesc_8c_source.html#l00197

Reply:
ah my bad, nice catch @daniel-jodlos. Will fix it.